ui/app/templates/components/forbidden-message.hbs

```diff
@@ -17,6 +17,12 @@
         requisite
       {{/if}}
       permission to view this.
+      <br>
+      <br>
```

Comment:
Woah, is this the first instance of two paragraphs of text in the UI? I couldn't find any prior art for this type of spacing.

What do you think about changing `empty-message-body` to being a `div` and then making each line a `p` and then creating this paragraph spacing with margins instead?

Something like this in the `.empty-message-body` selector:

```scss
p:not(:last-child) {
  margin-bottom: 0.5rem; // idk how much spacing this should actually be, somewhere between 0.5 and 1.
}
```

Reply:
I did find [this two-paragraph](https://github.com/hashicorp/nomad/blob/a82697a2f024555cd68d0fa4ba5b52257d885e6b/ui/app/templates/components/fs/file.hbs#L31-L37) empty message, though it’s slightly different in that the second paragraph is a button. In 412da49 I added this within `.empty-message-body`:

```
    &:not(:last-child) {
      margin-bottom: 1rem;
    }
 ```

and removed the unsightly `<br>`s, thanks for the push there. Here are screenshots of these two two-paragraph messages, does it seem suitable to you?

<img width="1176" alt="double-p-existing" src="https://user-images.githubusercontent.com/43280/113318927-91b2c180-92d6-11eb-82c3-c3f65bf5f118.png">

<img width="1189" alt="double-p-terminal" src="https://user-images.githubusercontent.com/43280/113318947-94151b80-92d6-11eb-8099-080864eba1fb.png">

Reply:
💯